documentation/modules/ROOT/pages/payloads.adoc

```diff
@@ -174,6 +174,100 @@ A Property `errorMessages` with a Collection Characteristic and effective data t
 }
 ----
 
+[[characteristics-payload-mappings]]
+== Specific payload mappings for Characteristics
+
+This section describes how data payloads look like for specific xref:characteristics.adoc[Characteristics].
+
+|===
+| Characteristic Name | Corresponding JSON data type
+| `samm-c:Either` | A JSON object with the left field and the value, which is one of the types proposed in `xref:characteristics.adoc#either-characteristic[Either]`.
+|===
+
+
+Example for `samm-c:Either`, a Property `speedProperty` has Characteristic and effective data type `samm-c:Either` and the possible values
+`ErrorMessage` (`xref:characteristics.adoc#single-entity-characteristic[Entity]` data type), which shows why speed can't be retrieved, or '60' (`xsd:integer`) would be serialized in the JSON payload as follows:
+
+Example:
+
+[source,turtle,subs="attributes+"]
+----
+include::example$aspect-with-either-characteristic.ttl[tags=aspect-with-either-characteristic]
+----
+
+Generated payload:
+
+[source,json]
+----
+{
+  "speedProperty" : {
+    "left" : {
+      "errorDescription" : "yedUsFwdkelQbxeTeQOvaScfqIOOmaa",
+      "errorCode" : "eOMtThyhVNLWUZNRcBaQKxI"
+    }
+  }
+}
+----
+
+or
+
+[source,json]
+----
+{
+  "speedProperty": {
+    "left": 60
+  }
+}
+----
```

Comment:
`samm-c:Either` is not a data type (!), and neither is `samm-c:SingleEntity`, therefore I'd change the introductory sentence accordingly. Additionally, after reviewing the existing section [Declaring Either](https://eclipse-esmf.github.io/samm-specification/snapshot/modeling-guidelines.html#declaring-either), I've seen that it already provides an example and I think it would make sense to refer to that same example here, i.e., remove aspect-with-either-characteristic.ttl (and if necessary adapt aspect-with-double-either-characteristic.ttl accordingly).

```suggestion
The following example Aspect uses `samm-c:Either` to express that the Property `speedValue` can either return the integer value for speed, or an Error message (using an Entity), if the value could not be determined (cf. xref:modeling-guidelines.adoc#declaring-either[Declaring Either]).

So, assuming an Aspect Model has a Property `result` with its `samm:characteristic` set to `:Result` according to the following definition:

[source,turtle,subs="attributes+"]
----
include::example$declaring-either.ttl[tags=either-declaration]
----

the corresponding JSON payload either contain a `left` key with the error description:

[source,json]
----
{
  "result": {
    "left": {
      "errorCode": "...",
      "errorDescription": "..."
    }
  }
}
----

or it can contain a `right` key with the success status:

[source,json]
----
{
  "result": {
    "right": {
      "status": "..."
    }
  }
}
----

Reply:
Agree that 'Either' is not a datatype. I will fixed it. But also, having another one example here is good idea, because we don't disturbing user to open new page with example. But from another side, yes, we don't repeat the same docs.

Thanks!